src/com/ka/spreadsheet/diff/Flags.java

```diff
@@ -9,19 +9,27 @@
   private static final String DIFF_NUMERIC_PRECISION_FLAG = "--diff_numeric_precision";
   // no value, default absent
   private static final String DIFF_IGNORE_FORMULAS_FLAG = "--diff_ignore_formulas";
+  private static final String DIFF_FORMAT_FLAG = "--diff_format";
 
   public static boolean DEBUG;
   public static Double DIFF_NUMERIC_PRECISION;
   public static boolean DIFF_IGNORE_FORMULAS;
+  public static DiffFormatter DIFF_FORMAT;
   public static File WORKBOOK1;
   public static File WORKBOOK2;
   public static WorkbookIgnores WORKBOOK_IGNORES1;
   public static WorkbookIgnores WORKBOOK_IGNORES2;
 
+  public enum DiffFormatter {
+    EXCEL_CMP,
```

Comment:
I really didn't know what to call your existing format, so I just used something simple.  Feel free to suggest an alternative.

Reply:
LGTM